Cargo.toml

```diff
@@ -127,3 +122,5 @@ getrandom = ["rand_core/getrandom"]
 # See https://github.com/rust-lang/cargo/issues/9210
 # and https://github.com/w3f/schnorrkel/issues/65#issuecomment-786923588
 
+[patch.crates-io]
+curve25519-dalek = { git = "https://github.com/mobilecoinfoundation/curve25519-dalek.git", rev = "8791722e0273762552c9a056eaccb7df6baf44d7" }
```

Comment:
This can be removed when https://github.com/dalek-cryptography/curve25519-dalek/pull/383/files is published